lang/pt-BR.json

```diff
@@ -0,0 +1,58 @@
+{
+    "CN": {
+        "SETTINGS": {
+            "DEFAULT_ICON.NAME": "Ícone Personalizado",
+            "DEFAULT_ICON.HINT": "Caminho para uma imagem aqui substituirá o ícone padrão usado para o efeito de concentração.",
+
+            "ITEM_ICON.NAME": "Usar Ícone do Item",
+            "ITEM_ICON.HINT": "Se ativado, a imagem usada para efeitos de concentração será substituída pela imagem do item.",
+
+            "PREPEND.NAME": "Prepend Effect Labels",
+            "PREPEND.HINT": "If checked, concentration effects are prepended with 'Concentration' to distinguish them from other effects.",
```

Comment:
I've had a look-over and this all looks good. I'd be willing to merge if you could translate these missing strings as well.

Reply:
I forgot haha
I skipped at first to look into the word prepend. Hold on.